environment.yml

```diff
@@ -0,0 +1,15 @@
+name: flexflow
```

Comment:
should this file go to the `conda` folder?

Reply:
Thanks! I hadn't seen the `conda` folder. It looks somewhat like a conda-forge feedstock (https://github.com/conda-forge/feedstocks). Something like https://github.com/conda-forge/unzip-feedstock/tree/main/recipe. It could be used for making flexflow installable from conda as `conda install flexflow -c conda-forge`. Though I could be completely wrong here.

> should this file go to the `conda` folder?

I am not sure about this. Maybe @eddy16112 can provide some context/let us know where we should put the `environment.yml` file?

Reply:
I guess this can be fixed after Wei's python PR is merged and when we add support for conda packaging of FlexFlow